src/main/java/org/betonquest/betonquest/modules/logger/custom/chat/PlayerFilter.java

```diff
@@ -0,0 +1,55 @@
+package org.betonquest.betonquest.modules.logger.custom.chat;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.logging.Level;
+
+/**
+ * This interface provides methods to manage filters for a player.
+ */
+public interface PlayerFilter {
+    /**
+     * Adds a filter to a player.
+     *
+     * @param uuid   The {@link UUID} of the player
+     * @param filter The filter pattern
+     * @param level  The {@link Level} of the filter
+     * @return True if the filter was successfully added
+     */
+    boolean addFilter(UUID uuid, String filter, Level level);
+
+    /**
+     * Removes a filter from a player.
+     *
+     * @param uuid   The {@link UUID} of the player
+     * @param filter The filter pattern
+     * @return True if the filter was successfully removed
+     */
+    boolean removeFilter(UUID uuid, String filter);
+
+    /**
+     * Gets a players filters.
+     *
+     * @param uuid The {@link UUID} of the player
+     * @return A list of filters
+     */
+    List<String> getFilters(UUID uuid);
```

Comment:
The result should probably be a `Set` as presence / absence checks are done often, order doesn't matter and multiple declaration of the same filter don't affect the result.